assets/src/edit-story/components/panels/document/slug/slug.js

```diff
@@ -38,24 +43,39 @@ export const MIN_MAX = {
   },
 };
 
-const BoxedTextInput = styled(TextInput)`
-  padding: 6px 6px;
-  border-radius: 4px;
-  flex-grow: 1;
-  &:focus {
-    background-color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.white};
+const PermalinkRow = styled(Row)`
+  margin-bottom: 12px;
+
+  * input {
+    color: ${({ theme }) => theme.colors.fg.tertiary};
```

Comment:
Designs have the input text as `tertiary` when it's filled.

Reply:
Curious why use `* input` instead of just `input`?

Reply:
oops, we don't need the `*` 😓